src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,23 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+
+interface Todo {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+}
+
+interface TodoListProps {
+  todos: Todo[];
+}
```

Comment:
You can move it to another file, and then export. ( Try to use types outside of the component file)